src/services/getCourtCase.ts

```diff
@@ -1,15 +1,14 @@
 import { DataSource, EntityManager } from "typeorm"
 import CourtCase from "./entities/CourtCase"
 import PromiseResult from "../types/PromiseResult"
-import courtCasesByVisibleForcesQuery from "./queries/courtCasesByVisibleForcesQuery"
 
-const getCourtCase = (
+const getCourtCase = async (
   dataSource: DataSource | EntityManager,
-  courtCaseId: number,
-  forces: string[]
+  courtCaseId: number
 ): PromiseResult<CourtCase | null> => {
-  const courtCaseRepository = dataSource.getRepository(CourtCase)
-  const query = courtCasesByVisibleForcesQuery(courtCaseRepository, forces)
+  const repository = dataSource.getRepository(CourtCase)
+  const query = repository
+    .createQueryBuilder("courtCase")
```

Comment:
Wouldn't we need to use the `courtCasesByVisibleForcesQuery` to ensure we show cases that are visible for the force?